yoda/llava_data_prep/table_templates/sec_edgar/AAPL.py

```diff
@@ -0,0 +1,189 @@
+synth_tables = []
```

Comment:
Similar comment to above. Could these live in a json file for the given company in this case Apple? That way they can be pulled in when necessary?